tunix/rl/agentic/prompts/prompt_example.yaml

```diff
@@ -0,0 +1,23 @@
+# Prompt example
```

Comment:
do you need to define it as yaml? what about define it as prompt_template.py with constant variable?